test/python/test_schemas.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+# pylint: disable=redefined-builtin
+
+""" Schemas test"""
+import json
+import os
+from qiskit import (validate_json_against_schema, _create_validator)
+from qiskit import __path__ as qiskit_path
+from .common import QiskitTestCase
+
+
+class TestSchemaExamples(QiskitTestCase):
+    """
+    Tests schema validation
+    """
+
+    def setUp(self):
+        self.schema_tests = []
```

Comment:
Although explicit, this code is a little bit messy. The use of nested `subTest` does not help either. What if replacing this with a class attribute `_json_examples_per_schema` (or similar) with a dictionary of pairs (name of the schema, list of examples):

```python
_json_examples_per_schema = {
  'backend-configuration': [
    'backend_configuration_openpulse_example.json',
    'backend_configuration_openqasm_example.json',
    'backend_configuration_openqasm_simulator_example.json'
  ],
  'backend_properties': [
    'backend_properties_example.json'
  ],
  ...
}
```
What do you think?